### PR TITLE
Add warmPrefixTarget and enablePrefixDelegation support to vpc cni op…

### PR DESCRIPTION
**Pull request by davidroth**
See: https://github.com/pulumi/pulumi-eks/issues/608

### CODE REVIEW

**Comment by leezen** [comment]
@davidroth Sorry for not replying sooner. Since this is a multi-language component, you should be able to automatically generate the appropriate SDKs. Note that the schema is actually generated from a Go program (see `provider/cmd/pulumi-gen-eks`) so if you modify that along with modifying the underlying provider itself (as you've done w/ the .ts files), then you should be able to run `make build` to generate the other language SDKs as part of this PR.

**Comment by davidroth** [comment]
@leezen Ok, I updated `provider/cmd/pulumi-gen-eks\main.go` and generated the files for the `dotnet` `python` and `go` providers.
Please let me know if anything is missing.

**Comment by davidroth** [comment]
Rebased, fixed linting error + updated changelog.

**Comment by davidroth** [comment]
@lblackstone Any chance of somebody reviewing this so that it can go into the 0.36 release?

**Comment by lblackstone** [comment]
> @lblackstone Any chance of somebody reviewing this so that it can go into the 0.36 release?

Thanks for the PR! I just added it to our review queue so we'll take a look shortly.

File: nodejs/eks/cmd/provider/cni.ts [hunk 1]
```diff
@@ -71,6 +73,12 @@ function computeVpcCniYaml(cniYamlText: string, args: VpcCniInputs): string {
     if (args.warmIpTarget) {
         env.push({name: "WARM_IP_TARGET", value: args.warmIpTarget.toString()});
     }
+    if (args.warmPrefixTarget) {
+        env.push({name: "WARM_PREFIX_TARGET", value: args.warmPrefixTarget.toString()});
+    }
+    if (args.enablePrefixDelegation) {
+        env.push({name: "ENABLE_PREFIX_DELEGATION ", value: args.enablePrefixDelegation.toString()});
```
Comment (lblackstone) [inline nodejs/eks/cmd/provider/cni.ts:80]: I think you need to handle the boolean value instead.
```suggestion
        env.push({name: "ENABLE_PREFIX_DELEGATION ", value: args.enablePrefixDelegation ? "true" : "false"});
```

**Review by lblackstone — CHANGES_REQUESTED** [review]
Looks good overall

**Review by lblackstone — APPROVED** [review]

**Pull request merged**